easybuild/easyconfigs/l/libosmium/libosmium-2.15.4-foss-2018b.eb

```
@@ -0,0 +1,46 @@
+easyblock = "CMakeMake"
+
+name = 'libosmium'
+version = '2.15.4'
+
```

We'll need to include a versionsuffix to indicate that this depends on python, even when it does to indirectly.
```suggestion
versionsuffix = '-Python-3.6.6'

```